llvm/utils/TableGen/ARMTargetDefEmitter.cpp

```diff
@@ -221,8 +221,28 @@ static void EmitARMTargetDef(RecordKeeper &RK, raw_ostream &OS) {
   OS << "#ifdef EMIT_CPU_INFO\n"
      << "inline constexpr CpuInfo CpuInfos[] = {\n";
 
+  std::map<std::string, std::pair<std::string, const Record *>> ProcessorModels;
```

Comment:
You don't really use the `std::string Alias` for anything but a comment, this could be
```cpp
 std::map<std::string, const Record *> ProcessorModels;
```
which is easier to understand.